packages/mwp-api-state/src/reducer.test.js

```diff
@@ -93,6 +106,17 @@ describe('getListState', () => {
 			},
 		});
 	});
+	it('merges new response with existing dynamicRef adding new items at the top of the list', () => {
+		const value = ['qux'];
+		expect(
+			getListState({ [resp.query.list.dynamicRef]: { value } }, respWithReverse)
+		).toEqual({
+			[resp.query.list.dynamicRef]: {
```

Comment:
It feels as if there is a way to simplify/consolidate the unit test code, especially in regard to all the various resp* objects. As an example, I find it odd that we're using `resp` versus `respWithReverse` for things like `dynamicRef` and `response.value`, but perhaps I'm overlooking a nuance. This new test seems to conform with the others & refactoring is probably outside the scope of this ticket, unless you want to & feel like taking a stab at improving the test quality here :)

Reply:
Totally agree. Looks like not a big change, so will update here.